src/_versions.ts

```diff
@@ -11,8 +11,8 @@ export interface TsAppVersion {
 export const versions: TsAppVersion = {
     version: '0.1.0',
```

Comment:
this version should be bumped up to 0.1.1

Reply:
done